dropwizard-request-logging/src/main/java/io/dropwizard/request/logging/filter/UrlFilterFactory.java

```diff
@@ -0,0 +1,43 @@
+package io.dropwizard.request.logging.filter;
+
+import ch.qos.logback.access.spi.IAccessEvent;
+import ch.qos.logback.core.filter.Filter;
+import ch.qos.logback.core.spi.FilterReply;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.dropwizard.logging.filter.FilterFactory;
+
+import java.util.Collections;
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+@JsonTypeName("url-filter-factory")
+public class UrlFilterFactory implements FilterFactory<IAccessEvent> {
+    @NotNull
+    private Set<String> urls = Collections.emptySet();
+
+    @JsonProperty
+    public Set<String> getUrls() {
+        return urls;
+    }
+
+    @JsonProperty
+    public void setUrls(final Set<String> urls) {
+        this.urls = urls;
+    }
+
+    @Override
+    public Filter<IAccessEvent> build() {
+        return new Filter<IAccessEvent>() {
+            @Override
+            public FilterReply decide(final IAccessEvent event) {
+                if (urls.contains(event.getRequestURI())) {
```

Comment:
Rename to uri to match up with the field from the event. Since url and uri are both available.

https://github.com/qos-ch/logback/blob/8fccc8879641be6c4ad7da0f2a6057f6bb9fbd47/logback-access/src/main/java/ch/qos/logback/access/spi/IAccessEvent.java#L83-L88